src/vs/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -3494,6 +3494,25 @@ export class InstallLocalExtensionsInRemoteAction extends Action {
 	}
 }
 
+export class OpenExtensionAuthoringDocsAction extends Action {
```

Comment:
Put this in https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/extensions/browser/extensionsActions.ts#L1